### PR TITLE
Use OPDSImporterWithS3Mirror instead of custom code

### PR DESCRIPTION
**Pull request by leonardr**
This branch takes advantage of the new OPDSImporterWithS3Mirror class to get rid of code for automatically mirroring incoming representations as part of an OPDS import.

I also move several scripts from `bin/` into `bin/repair` and `bin/util`--the same sort of work I've already done in circulation and metadata.

The `unglue_it_monitor` and `standard_ebooks_monitor` scripts still don't actually work reliably, but this is a step towards getting them to work reliably.

### CODE REVIEW

File: opds_import.py [hunk 1]
```diff
@@ -1,60 +1,22 @@
...
-
-class UnglueItOPDSImporter(ContentOPDSImporter):
+class UnglueItOPDSImporter(OPDSImporterWithS3Mirror):
     """Importer for unglue.it OPDS feed, which has acquisition links from multiple sources for some entries."""
 
-    def import_from_feed(self, feed):
-        return super(UnglueItOPDSImporter, self).import_from_feed(feed, even_if_no_author=True)
+    def import_from_feed(
+            self, feed, even_if_no_author=False, 
+            cutoff_date=None, immediately_presentation_ready=False
```
Comment (aslagle) [inline opds_import.py]: Why have the arguments here if they're always overridden?

Comment (leonardr) [inline opds_import.py]: This is common code called by OPDSImportMonitor.follow_one_link, so I couldn't change the signature. I've changed the defaults to reflect the values that this subclass uses.

I think that even_if_no_author, cutoff_date, and immediately_presentation_ready have become kind of miscellaneous and should go into a new Policy object (or even into ReplacementPolicy, I'm not sure). I think it's the same divide I mentioned earlier--either we are importing the actual books (in which case they should be made presentation ready ASAP even if there is no known author) or we are importing metadata (in which case we should hold off until we get all the data).

Does this answer your question?

Comment (aslagle) [inline opds_import.py]: :+1:

**Comment by aslagle** [comment]
Looks good :)

**Pull request merged**